orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -232,4 +238,25 @@ class OperationsController {
     def pipeline = orchestrationLauncher.start(json)
     [ref: "/tasks/${pipeline.id}".toString()]
   }
+
+  @RequestMapping(value = "/instance/enabled", method = RequestMethod.POST)
+  void setInstanceStatus(@RequestBody Map<String, Boolean> enabledWrapper) {
+    if (!discoveryStatusPoller) {
+      log.warn("No configured discovery status poller")
+      return
+    }
+
+    if (!discoveryStatusPoller instanceof  NoDiscoveryApplicationStatusPublisher) {
+      throw new IllegalStateException("Discovery status cannot be overwritten - there is an existing discovery client managing orca")
+    }
+
+    NoDiscoveryApplicationStatusPublisher noDiscoveryApplicationStatusPublisher = (NoDiscoveryApplicationStatusPublisher) discoveryStatusPoller;
+
+    boolean enabled = enabledWrapper.get("enabled")
```

Comment:
A boolean param alone wasn't being deserialized correctly/I don't know what content type makes sense there - do you have an example of this working?